pagopa/io-app-design-system
# feat: [IOBP-301] Add new icons `docPaymentCode`, `notes` and `entityCode`

## Short description
This PR adds three new icons:
- `docPaymentCode`
- `notes`
- `entityCode`

## List of changes proposed in this pull request
- Added originals svg into `originals` folder;
- Added react components with svg inside;

## How to test
Open the example app and navigate to the icons and check that these icons showed into [this figma](https://www.figma.com/file/KY9BpP4LaSSd18fuYMwxYK/Paga-un-avviso-pagoPA?type=design&node-id=11-930&mode=design&t=H5KE7JSSAS4vzqkX-4) are present.

## Preview cons added
<img width="59" alt="image" src="https://github.com/pagopa/io-app-design-system/assets/34343582/bb40758e-d443-474b-88aa-c2aaefa404c7">
<img width="47" alt="image" src="https://github.com/pagopa/io-app-design-system/assets/34343582/02474b1c-1aff-44bb-a06f-445b86d23716">
<img width="43" alt="image" src="https://github.com/pagopa/io-app-design-system/assets/34343582/cff1cc47-072d-48ee-a906-6054755ce88c">
